app/templates/ratemypdf_stats.html

```diff
@@ -236,6 +277,161 @@ <h2 class="pdf-section-heading">Suggestions</h2>
+          {% endif %}
+          {% if stats.get("description") and stats["description"].lower().strip() not in ["abortthisnow.", "apierror"] %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-description">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-description" aria-expanded="false" aria-controls="collapse-description">
+                Suggested plain language description
+              </button>
+            </h2>
+            <div id="collapse-description" class="accordion-collapse collapse" aria-labelledby="heading-description" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                {{ stats.get("description") }}
+              </div>
+            </div>
+          </div>
+          {% endif %}
+
+          {% if stats.get("passive sentences", []) %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-passive-voice">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-passive-voice" aria-expanded="false" aria-controls="collapse-passive-voice">
+                Sentences with passive voice
+              </button>
+            </h2>
+            <div id="collapse-passive-voice" class="accordion-collapse collapse" aria-labelledby="heading-passive-voice" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                <ul>
+                {% for sentence in stats.get("passive sentences", []) %}
+                <li>
+                  {{ highlight_text(sentence[0], sentence[1])  | safe }}
+                </li>
+                {% endfor %}
+                </ul>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% if stats.get("difficult words", []) %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-difficult-words">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-difficult-words" aria-expanded="false" aria-controls="collapse-difficult-words">
+                Consider replacing difficult words
+              </button>
+            </h2>
+            <div id="collapse-difficult-words" class="accordion-collapse collapse" aria-labelledby="heading-difficult-words" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                We refer to [plainlanguage.gov](https://www.plainlanguage.gov/guidelines/words/)'s list of common
+                complex words and simple replacements. Apply judgement and read the words in context to make sure the replacements
+                still make sense.
+                <ul>
+                {% for word in stats.get("difficult words", []) %}
+                <li>
+                  {{ word  }}
```

Comment:
Not urgent, but it's weird that all the other categories we show the words in context, and this one we specifically say "make sure this makes sense in context", but don't show the context at all. I know we can't get word substitutions automatically, but showing the rest of the sentence would help.

Reply:
Good idea but it belongs in formfyxer and needs a separate issue. To me it's tied to getting a better list of simple words that accounts for words people need in legal forms.